spec/features/idv/analytics_spec.rb

```diff
@@ -850,20 +850,21 @@ def wait_for_event(event, wait)
         and_return(true)
       allow_any_instance_of(FederatedProtocols::Oidc).
         to receive(:biometric_comparison_required?).
-        and_return({ biometric_comparison_required: true })
+        and_return(true)
+
+      allow(IdentityConfig.store).to receive(:des).
+        and_return(true)
```

Comment:
what is _des_ ... i am unable to locate this? 🤔

Reply:
@amirbey , oooh, my IDE completion is too aggressive.